session.go

```diff
@@ -485,13 +501,17 @@ func (s *session) run() {
 
 func (s *session) handleLoop() {
 	var (
-		err    error
-		flag   bool
-		wsFlag bool
-		wsConn *gettyWSConn
-		// start  time.Time
-		counter gxtime.CountWatch
-		outPkg  interface{}
+		err      error
+		ok       bool
+		flag     bool
+		wsFlag   bool
+		udpFlag  bool
+		loopFlag bool
+		wsConn   *gettyWSConn
+		counter  gxtime.CountWatch
+		outPkg   interface{}
+		pkgBytes []byte
+		iovec    [][]byte
```

Comment:
io vector ?

Reply:
yes. If go1.13 stable version is released, I will use writev in place of session.WriteBytesArr.